pandas/core/window/expanding.py

```diff
@@ -148,8 +148,23 @@ def count(self, **kwargs):
 
     @Substitution(name="expanding")
     @Appender(_shared_docs["apply"])
-    def apply(self, func, raw=False, args=(), kwargs={}):
-        return super().apply(func, raw=raw, args=args, kwargs=kwargs)
+    def apply(
+        self,
+        func,
+        raw=False,
+        engine="cython",
+        engine_kwargs=None,
```

Comment:
Can you add annotation for this? I assume `Dict[str, Any]` unless can further subset the provided values